### ISSUE TITLE
TeamSpeak3QueryApi.pdb not loaded

### ISSUE THREAD

**Issue by Judasz121** [nikeee/TeamSpeak3QueryAPI#54]
I ve installed this API with NuGet, and tried to use the code example to connect to TS3Query plugin
```
TeamSpeakClient rc = new TeamSpeakClient("localhost", 25639);
await rc.Connect();
await rc.UseServer(1);
var me = await rc.WhoAmI();
```
After executing this code i get a screen saying that TeamSpeak3QueryApi is not loaded
![image](https://user-images.githubusercontent.com/56927621/85729335-02e69800-b6f9-11ea-9e12-12b885f878cb.png)
I`m a beginner so i dont know much about C# but from what i ve searched the .pdb files are used in debugging - am I supossed to apply some special debbuging settings? I ve tried few and none of them worked.
I dont know if I`m doing something wrong or this .pdb file really should be there?

**Comment by nikeee** [comment]
I think the nuget package is currently shipped without the pdb. We can change that.

**Comment by nikeee** [comment]
I tried to fix this in version `1.12.0`. Please comment if the issue persists.
